superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1207,7 +1207,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
           required
           validationMethods={{ onBlur: () => {} }}
           errorMessage={validationErrors?.confirm_overwrite}
-          label={t(`TYPE "OVERWRITE" TO CONFIRM`)}
+          label={t('Type "%s" to confirm', t('OVERWRITE'))}
```

Comment:
I'm thinking you're right to not have the whole sentence in uppercase here, but pinging @jess-dillard and @kasiazjc in case there's a use case for "yelling" that's not covered in the [Caplitalization Guidelines](https://github.com/apache/superset/wiki/Capitalization-Guidelines).

Reply:
Currently 2 of 3 "confirm" messages are not in uppercase, so I thought it would be right to achieve 3 of 3 :)

![image](https://user-images.githubusercontent.com/53895552/214798356-95fedcf3-e49d-494e-ad57-eb9c796fb83c.png)